entity-service/src/main/kotlin/com/egm/stellio/entity/model/Attribute.kt

```diff
@@ -28,6 +28,8 @@ open class Attribute(
     @JsonIgnore
     var modifiedAt: ZonedDateTime? = null,
 
+    var datasetId: String? = null,
```

Comment:
`datasetId` is supposed to be of URI type. It is a good time to start using this (we should have done it for id). Does neo4j have a type for URI?

Reply:
As I see [here](https://neo4j.com/docs/cypher-manual/current/syntax/values/), there is no support for URI data type.
Seeing the spatial type Point makes me wonder about the other spatial type that we could have.

Reply:
Ok, so we could have it as an URI in the code and persisted as a string?

Reply:
Yes